packages/frontend/src/retro/components/dialogs/CreateTimerDialog.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button, Dialog, DialogContent, DialogTitle, DialogActions } from "@mui/material";
+import { CallToActionButton } from "../../../common/components/buttons/CallToActionButton";
+import { DialogProps } from "../../../common/types/commonTypes";
+import { useFullscreen } from "../../hooks/useFullscreen";
+import { TimePicker } from "../../../common/components/TimePicker";
+import React = require("react");
+
+interface CreateTimerDialogProps extends DialogProps {
+  startTimer: (duration: number) => void;
+  stopTimer: () => void;
+  pauseTimer: () => void;
+  isTimerRunning: boolean;
+  remainingMinutes: number;
+  remainingSeconds: number;
+}
+export function CreateTimerDialog({
+  isOpen,
+  close,
+  startTimer,
+  stopTimer,
+  pauseTimer,
+  isTimerRunning,
+  remainingMinutes,
+  remainingSeconds,
+}: CreateTimerDialogProps) {
+  const fullScreen = useFullscreen();
+
+  function handleSubmit() {
+    close();
+    if (isTimerRunning) {
+      stopTimer();
+    } else {
+      startTimer(100000);
```

Comment:
Magic number. Please define a variable in this file and give it a name describing what this number is.

Reply:
Do I need to give it a name, it is currently a placeholder value and should finally be replaced by the timepicker value set by the user. I could mark it as non permanent ?

Reply:
Let's check again once this is resolved.

Reply:
The timer now starts with the picked time, so this issue is resolved